linkerd/app/integration/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "linkerd2-app-integration"
+version = "0.1.0"
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
+edition = "2018"
+publish = false
+description = """
+Proxy integration tests
+
+The test utilities can be very costly to compile, so they are extracted into
+a dedicated crate to help the compiler cache dependencies properly.
+"""
+
+[features]
+# Disable to skip certain tests that should not be run on CI.
+flaky_tests = []
+
+[dependencies]
+bytes = "0.4"
+futures = "0.1"
+h2 = "0.1"
+http = "0.1"
+http-body = "0.1"
+hyper = "0.12"
+linkerd2-app = { path = ".." }
+linkerd2-app-core = { path = "../core" }
+linkerd2-metrics = { path = "../../metrics", features = ["test_util"] }
+linkerd2-task    = { path = "../../task", features = ["test_util"] }
```

Comment:
Extremely nitpicky: it looks like some whitespace was added here to align it with a dependency on another line, but the rest of the file is not aligned...maybe remove the whitespace?